lib/tasks/users.rake

```diff
@@ -33,4 +33,29 @@ namespace :users do
     remaining = User.where.not(latitude: nil, longitude: nil).where(state: nil).count
     p "#{remaining} users left to be updated.  Task can be reran in 24 hours."
   end
+
+  desc "Tags select users as Community Leaders"
+  task seed_leader: :environment do
+    users = [ {id: 4, first_name: 'Nell', last_name: 'Shamrell-Harrington' }, {id: 3, first_name: 'Rick', last_name: 'Rein'}]
```

Comment:
1.  These hashes would not have an `id` key/value pair.  Just first name and last name.
2.  I know this list is for development; reminder, we will need the **actual** list of users in this rake task before merging the PR.